### ISSUE TITLE
Unable to create account

### ISSUE THREAD

**Issue by Strikerrr** [Cvolton/GMDprivateServer#176]
"Something went wrong"

**Comment by Strikerrr** [comment]
Determined that it's FWHA being shitty but can anyone advise on this?

**Comment by EndNation** [comment]
@Strikerrr Like I said, just host it on your own network, easier to do alot of stuff :)

**Comment by Strikerrr** [comment]
@EndNation yes, but how would I set up the whole database stuff and FTP and such? It's all complicated to me xD

**Comment by EndNation** [comment]
Just use wamp and stuff
